From e06bbd3f0ba3a8f18da91b4e361aac05ff5b90f2 Mon Sep 17 00:00:00 2001
From: Richard Oliver <richard.oliver@raspberrypi.com>
Date: Thu, 20 Jun 2024 09:58:32 +0100
Subject: [PATCH 508/697] lib: earlycpio: export symbol find_cpio_data()

Add EXPORT_SYMBOL_GPL() for find_cpio_data() so that loadable modules
may also parse uncompressed cpio.

Signed-off-by: Richard Oliver <richard.oliver@raspberrypi.com>
---
 lib/earlycpio.c | 1 +
 1 file changed, 1 insertion(+)

--- a/lib/earlycpio.c
+++ b/lib/earlycpio.c
@@ -139,3 +139,4 @@ struct cpio_data find_cpio_data(const ch
 quit:
 	return cd;
 }
+EXPORT_SYMBOL_GPL(find_cpio_data);
